packages/grid/_modules_/grid/models/api/localeTextApi.ts

```diff
@@ -40,6 +40,17 @@ export interface LocaleText {
   filterPanelOperatorOr: React.ReactNode;
   filterPanelColumns: React.ReactNode;
 
+  // Filter operators text
+  contains: React.ReactNode;
```

Comment:
These can only be strings:

```suggestion
  contains: string;
```

https://codesandbox.io/s/objective-dewdney-rtyxo?file=/src/App.tsx:0-150

x the other keys